packages/core/stories/tokens/typography/Typography.stories.mdx

```diff
@@ -102,13 +102,13 @@ If your project needs to support an addtional language, please [submit a feature
 #### Armenian script
 
 <p className="lang-label">Armenian: </p>
-<p lang="hy" className="font-specimen">Ու նայիրեան աղջիկների հեզաճկուն պա՛րն եմ սիրում։</p>
+<p lang="hy" className="font-specimen">Բարի գալուստ Մասաչուսեթս</p>
```

Comment:
My original comment was about reverting all the font specimen text changes in this file, not just for these two languages

Reply:
The specimen text should match what's in production --> https://mayflower.digital.mass.gov/core/index.html?path=/docs/foundation-typography--page